Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM buildpack-deps:bionic
+FROM ubuntu:18.04
```

Comment:
Does 22.04 fail? I remember there was an issue with upgrading in the past, but the details escape me.

Reply:
Yes, it does, and it seems that it has something to do with the fact that the toolset uses Python 2. Given that the plan (#6064) is to replace the current toolset with one rewritten from scratch, I have decided not to bother with it. While this container is not that critical with respect to performance as the database and the wrk ones, during the first time a developer runs the `tfb` script the contrast between `buildpack-deps` and `ubuntu` is quite stark, so IMHO this change is still worth it while the toolset transition has not been completed, especially for people with slower Internet connections.